server/game/cards/10-SoD/SpareBoot.js

```diff
@@ -0,0 +1,33 @@
+const DrawCard = require('../../drawcard.js');
+
+class SpareBoot extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Stand and move attachment',
+            phase: 'challenge',
+            limit: ability.limit.perPhase(1),
+            target: {
+                activePromptTitle: 'Select an attachment',
+                cardCondition: card => card.location === 'play area' && card.getType() === 'attachment' &&
+                                       card.parent && card.parent.controller === this.controller &&
+                                       (card.parent !== this || card.kneeled) &&
+                                       this.controller.canAttach(card, this)
```

Comment:
This needs to check `card.controller.canAttach(card, this)` instead. Otherwise I believe moving an "opponent only" attachment will fail because it will use the current player as the potential controller and expect it to be attached to a player other than the controller of Spare Boot.